src/Qwiq.Core.Rest/Project.cs

```diff
@@ -64,7 +68,27 @@ internal Project([NotNull] TeamProjectReference project, [NotNull] WorkItemStore
                                               .GetResult();
 
                             return new Node(result).ChildNodes;
-                        }))
+                        }),
+                new Lazy<IQueryFolderCollection>(() =>
+                {
+                    return new QueryFolderCollection(() =>
+                    {
+                        //BUGBUG: There is a bug in the GetQueryAsync in vsts where if a folder contains a '+' character it will return 404, even if the folder exists see here: https://developercommunity.visualstudio.com/content/problem/123660/when-a-saved-query-folder-contains-a-character-it.html
+                        QueryHierarchyItem FolderExpansionFunc(QueryHierarchyItem item)
```

Comment:
Why does this need to be passed into the ctor? Can we encapsulate it in the Rests's `QueryFolder` object? Unclear why `Project` would need to know this information

Reply:
I'll revisit